### PR TITLE
Unify backstage styling

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
A handful of changes to align with mocks:
* Update the incident details header to feature the same styling as the playbook edit screen's header, notably without showing the top-level "Playbooks | Incidents" navigation.
* Remove the "back arrow" from the main list view, since we are expecting the user to use the `X` on the right instead.
* Remove the `X` from the playbook edit screen.
* Update to `Export Channel Log` nomenclature.

Playbook list:
image

Playbook details:
image

Incident list:
image

Incident details:
image

### CODE REVIEW

File: webapp/src/components/backstage/incidents/incident_details/incident_details.tsx [hunk 1]
```diff
@@ -14,46 +15,246 @@ import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
...
+    .profile {
+        font-size: 14px;
+    }
+`;
+
+const NavbarPadding = styled.div`
+    flex-grow: 1;
+`;
+
+const BackstageIncidentDetailsContainer = styled.div`
```
Comment (lieut-data) [inline webapp/src/components/backstage/incidents/incident_details/incident_details.tsx:72]: I didn't break this up into discrete components -- for now.

File: webapp/src/components/backstage/incidents/incident_details/incident_details.tsx [hunk 2]
```diff
@@ -14,46 +15,246 @@ import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
...
-        try {
-            setIncident(await fetchIncidentWithDetails(incidentId));
-            setFetchingState(FetchingStateType.fetched);
-        } catch {
-            setFetchingState(FetchingStateType.notFound);
-        }
-    };
-
     useEffect(() => {
+        const fetchIncident = async (incidentId: string) => {
```
Comment (lieut-data) [inline webapp/src/components/backstage/incidents/incident_details/incident_details.tsx:249]: Tidying up some tslint warnings.

File: webapp/src/components/backstage/incidents/status_badge.tsx [hunk 3]
```diff
@@ -3,16 +3,31 @@
...
+    background-color: var(--center-channel-color-16);
+    font-weight: 600;
 
-    return (
-        <span className={badgeClass}>
-            {badgeText}
-        </span>
-    );
-}
+    &.ongoing {
```
Comment (lieut-data) [inline webapp/src/components/backstage/incidents/status_badge.tsx:21]: I think I'm supposed to use `props` here, but then I'd have to define a type... so I cheated.

**Review by asaadmahmood — APPROVED** [review]
Looks great!

File: webapp/src/components/backstage/incidents/incident_details/incident_details.tsx (hunk 1 above)
Comment (cpoile) [reply webapp/src/components/backstage/incidents/incident_details/incident_details.tsx:72]: The only thing I'm annoyed with (with styled components) is that my IDE doesn't highlight it and autocomplete it, like it does for css or sass files. :(

**Review by cpoile — APPROVED** [review]

**Pull request merged**